spec/controllers/application_controller_spec.rb

```diff
@@ -1,140 +1,141 @@
 require 'rails_helper'
 
 RSpec.describe ApplicationController, controller: true do
-  describe '#require_authentication (via API)' do
```

Comment:
These tests need to be completely rewritten because they mock internals that are no longer there. Maybe get rid of this and replace with functional tests of the authentication